### PR TITLE
[Feature][Merv] Add google button & sign-in/register layout

### PR DESCRIPTION
**Pull request by jdesolate**

### CODE REVIEW

File: components/CollapsedBar/index.tsx [hunk 1]
```diff
@@ -34,34 +34,39 @@ function CollapsedBar(props: Omit<NavbarProps, "children"> & Props) {
                     icon={<IconSpaceDashboard />}
                     label={"Dashboard"}
                     onClick={() => setPage(1)}
+                    isFullWidth={true}
```
Comment (tapsTapayan) [inline components/CollapsedBar/index.tsx]: @jdesolate include nalang tanan ni follow

File: components/IconButton/index.tsx [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
+import React, { MouseEventHandler } from "react";
 
 import style from "./_index.module.scss";
 
 type Props = Omit<LinkProps, "href"> & {
     icon: React.ReactNode;
     label: string;
-    className?: "link" | "active";
+    className?: "link" | "active" | "back";
+    onClick?: MouseEventHandler<HTMLButtonElement>;
```
Comment (tapsTapayan) [inline components/IconButton/index.tsx]: @jdesolate LinkProps has already onClick with it

File: components/StyledButton/index.tsx [hunk 3]
```diff
@@ -12,13 +12,14 @@ type Button_Params = {
         | "submit"
         | "card"
         | "navigation";
+    onClick?: MouseEventHandler<HTMLButtonElement>;
```
Comment (tapsTapayan) [inline components/StyledButton/index.tsx]: use LinkProps instead @jdesolate

File: components/TextField/index.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { TextInput } from '@mantine/core';
+import style from "./_index.module.scss";
+
+type Props = {
+  label: string;
+  placeholder: string;
+}
```
Comment (tapsTapayan) [inline components/TextField/index.tsx]: use TextInputProps with Omit lang @jdesolate

File: pages/storypage.tsx [hunk 5]
```diff
@@ -34,13 +35,15 @@ function StoryPage() {
                 className="link"
                 icon={<IconHolidayVillage />}
                 label={"Link Inactive"}
+                isFullWidth={true}
```
Comment (tapsTapayan) [inline pages/storypage.tsx]: @jdesolate kani pud

File: pages/storypage.tsx [hunk 6]
```diff
@@ -34,13 +35,15 @@ function StoryPage() {
...
             />
             <IconButton
                 className="active"
                 icon={<IconHolidayVillage />}
                 label={"Link Active"}
+                isFullWidth={true}
```
Comment (tapsTapayan) [inline pages/storypage.tsx]: @jdesolate

**Review by tapsTapayan — CHANGES_REQUESTED** [review]
Done, mark lang resolved ang comments nya lahos nalang merge

File: components/CollapsedBar/index.tsx (hunk 1 above)
Comment (jdesolate) [reply components/CollapsedBar/index.tsx]: resolved

File: components/IconButton/index.tsx (hunk 2 above)
Comment (jdesolate) [reply components/IconButton/index.tsx]: resolved

File: components/StyledButton/index.tsx (hunk 3 above)
Comment (jdesolate) [reply components/StyledButton/index.tsx]: resolved

File: components/TextField/index.tsx (hunk 4 above)
Comment (jdesolate) [reply components/TextField/index.tsx]: resolved

File: pages/storypage.tsx (hunk 5 above)
Comment (jdesolate) [reply pages/storypage.tsx]: resolved

File: pages/storypage.tsx (hunk 6 above)
Comment (jdesolate) [reply pages/storypage.tsx]: resolved

**Review by tapsTapayan — APPROVED** [review]

**Pull request merged**